src/app/components/search-page/search-page.component.html

```diff
@@ -4,17 +4,22 @@
   </div>
 
   <div *ngFor="let item of result">
-    <div class="d-flex flex-column mb-2">
-      <img class="align-self-center" id="image-container" [src]="item.imageurl" />
+    <div>
+      <img class="item-img w-100 h-100 p-1" [src]="item.imageUrl" alt="Picture">
     </div>
 
-    <div class="d-flex align-items-center">
-      <div class="font-weight-bold">{{item.userdisplayname}}</div>
+    <div class="d-flex align-items-center my-2" *ngIf="item">
+      <fa-icon *ngIf="item.photoURL === 'undefined'" [icon]="faUserCircle" class="mr-3 ml-2 my-auto"></fa-icon>
+      <img *ngIf="item.photoURL !== 'undefined'" [src]="item.photoURL" alt="profile picture"
+        class="rounded-circle image-size mr-3 ml-2 my-auto">
```

Comment:
I think we can refactor the above ngIf logic to: 
!item.photoURL
item.photoURL